### PR TITLE
Profile page update

### PR DESCRIPTION
**Pull request by pomoalex**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -214,6 +236,159 @@ h5 {
...
+
+.big {
+	font-size: larger;
+}
+
+.bigger {
+	font-size: x-large;
+}
+
+.grey-box {
```
Comment (bogsen) [inline style.css]: These 2 classes don't seem to be related to fonts

File: profile.html [hunk 2]
```diff
@@ -20,23 +20,25 @@
...
+					<div id="profile-box-grid">
+						<div>
+							<img id="profile-picture" src="images/geo.jpg" alt="Profile picture">
+						</div>
+						<div id="brief-profile-description">
+							<span class="bigger">Georgian</span> <br>
+							<span class="bigger">Popescu</span> <br>
+							<small>here since Mar 2018</small>
+						</div>
+							<div id="profile-description">
```
Comment (bogsen) [inline profile.html]: Indentation

**Review by bogsen — COMMENTED** [review]
👍🏻

File: style.css (hunk 1 above)
Comment (pomoalex) [reply style.css]: .big and .bigger can be used to replace the obsolete <big> html tag .

Comment (bogsen) [reply style.css]: .grey-box and .grid-container-two-columns

File: profile.html [hunk 3]
```diff
@@ -20,23 +20,25 @@
...
-					</div>
-						<div id="profile-description">
-						<span class="big">
-							Achievements : 3/10<br>
-							Courses : 1/3<br>
-							Time spent learning : 2 h<br>
-							Longest streak : 1 days<br>
-							Title : Newbie
-						</span>
+				<div class="grid-itemm">
```
Comment (bogsen) [inline profile.html]: Please choose a better name for this class

**Review by bogsen — DISMISSED** [review]

File: profile.html (hunk 3 above)
Comment (pomoalex) [reply profile.html]: Sorry , I forgot to .

File: style.css [hunk 4]
```diff
@@ -127,6 +139,7 @@ body > .container {
 	box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.15), 0 6px 20px 0 rgba(0, 0, 0, 0.15);
 }
 
+/* Course Page CSS */
```
Comment (bogsen) [inline style.css:142]: Newline missing

Comment (pomoalex) [reply style.css:142]: It will be fixed in a future update .

**Review by MGafitescu — APPROVED** [review]
All good.
